chatt-app/ChatClient.java

```diff
@@ -1,155 +0,0 @@
-import java.awt.event.ActionEvent;
-import java.awt.event.ActionListener;
-import java.awt.event.KeyEvent;
-import java.awt.event.KeyListener;
-import java.awt.event.WindowAdapter;
-import java.awt.event.WindowEvent;
-import java.io.*;
-import java.net.*;
-
-import javax.swing.*;
-
-

```

Comment:
Description: this is a chat client that will connect to the chat server that allows multiple chat clients to communicate with each other via a socket.